docs/api/0.21/feature-session/index-outline.html

```diff
@@ -157,7 +157,7 @@
 <link rel="stylesheet" href="style.css">
 </HEAD>
 <BODY>
-<a href="/Users/skaspari/Projects/Mozilla/android-components/build/javadoc/feature-session/index"><a href="mozilla.components.feature.session/-session-use-cases/-request-desktop-site/invoke.html"><span class="keyword">fun </span><span class="identifier">invoke</span><span class="symbol">(</span><span class="identifier" id="mozilla.components.feature.session.SessionUseCases.RequestDesktopSite$invoke(kotlin.Boolean, mozilla.components.browser.session.Session)/enable">enable</span><span class="symbol">:</span>&nbsp;<a href="https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-boolean/index.html"><span class="identifier">Boolean</span></a><span class="symbol">, </span><span class="identifier" id="mozilla.components.feature.session.SessionUseCases.RequestDesktopSite$invoke(kotlin.Boolean, mozilla.components.browser.session.Session)/session">session</span><span class="symbol">:</span>&nbsp;<span class="identifier">Session</span>&nbsp;<span class="symbol">=</span>&nbsp;sessionManager.selectedSessionOrThrow<span class="symbol">)</span><span class="symbol">: </span><a href="https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-unit/index.html"><span class="identifier">Unit</span></a></a></a><br/>
+<a href="/Users/skaspari/Projects/Mozilla/android-components/build/javadoc/feature-session/index"><a href="mozilla.components.feature.session/-session-use-cases/-request-desktop-site/invoke.html"><span class="keyword">fun </span><span class="identifier">invoke</span><span class="symbol">(</span><span class="identifier" id="mozilla.components.feature.session.SessionUseCases.RequestDesktopSiteUseCase$invoke(kotlin.Boolean, mozilla.components.browser.session.Session)/enable">enable</span><span class="symbol">:</span>&nbsp;<a href="https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-boolean/index.html"><span class="identifier">Boolean</span></a><span class="symbol">, </span><span class="identifier" id="mozilla.components.feature.session.SessionUseCases.RequestDesktopSiteUseCase$invoke(kotlin.Boolean, mozilla.components.browser.session.Session)/session">session</span><span class="symbol">:</span>&nbsp;<span class="identifier">Session</span>&nbsp;<span class="symbol">=</span>&nbsp;sessionManager.selectedSessionOrThrow<span class="symbol">)</span><span class="symbol">: </span><a href="https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-unit/index.html"><span class="identifier">Unit</span></a></a></a><br/>
```

Comment:
Oh, did the IDE update those files? We don't want to do that for past releases :)

Reply:
Thanks for catching that, I will undo those files